kymatio/scattering2d/backend/backend_skcuda.py

```diff
@@ -69,23 +69,25 @@ def __init__(self, pad_size, input_size, pre_pad=False):
         self.build()
 
     def build(self):
-        pad_size_tmp = self.pad_size
+        pad_size_tmp = list(self.pad_size)
```

Comment:
What is this for? What other than list/tuple/array can be expected as input here requiring this? An iterator?

Reply:
OH. it's supposed to make a copy? And you want to modify it lower down, so tuples are out

Reply:
Yes this is to make a copy. Besides, this means you now can pass tuples in, which I think doesn't hurt.